.github/workflows/kubectl-deploy-job-eks.yml

```diff
@@ -0,0 +1,91 @@
+name: Superlinter
```

Comment:
I imagine you want a different name here.

Reply:
You're right